.github/workflows/compatibility_sharded.yml

```diff
@@ -4,6 +4,7 @@ concurrency:
   group: compatibility-checker-cluster
 
 on:
+  pull_request:
```

Comment:
TODO: will remove after testing

Reply:
@williampsmith You can just add `workflow_dispatch` above the schedule and then you can kick it off manually.
see https://github.com/MystenLabs/sui-operations/blob/main/.github/workflows/staging-heartbeat.yml#L7-L9
No need to remove it later after you are done testing